src/components/OrgView/OrgViewApp.js

```diff
@@ -157,13 +153,13 @@ function App({
     })
 
     if (wrapper) {
```

Comment:
```suggestion
    if (wrapper && routing.mode.name === 'org') {
```

https://github.com/aragon/aragon/pull/1272#issuecomment-641276914 is caused by `setAppPath` firing after the context has updated and removed `instanceId` and `instancePath`.

Reply:
I think we could test `instanceId` and `instancePath`? https://github.com/aragon/aragon/pull/1272/commits/b38f082ad6840b3b7ab716afa6823cb5058099ce

Reply:
Yes this is better I think, it was my initial thought but wasn't sure if it was self documenting enough.